1-exercises/F-for-of-loop/exercise.js

```diff
@@ -11,6 +11,13 @@ let tubeStations = [
     "Tottenham Court Road"
 ];
 
+for (let i = 0 ; i < tubeStations.length; i++){
```

Comment:
This line and line 21 should be a for of loop. The keyword "of" should be used. Here are some examples https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of